Model/FilePayload.php

```diff
@@ -748,25 +748,27 @@ private function addContactToQueue()
     public function evaluateSchedule($prepFile = false)
     {
         if (!$this->scheduleStart && !$this->test) {
-            $rate     = max(1, (int) $this->settings['rate']);
-            $seekDays = 30;
+            $rate   = max(1, (int) $this->settings['rate']);
+            $endDay = 30;
             $this->scheduleModel
                 ->reset()
                 ->setContactClient($this->contactClient)
                 ->setTimezone();
 
-            list($start, $end) = $this->scheduleModel->nextOpening($rate, $seekDays);
-
-            if (!$start) {
+            $openings = $this->scheduleModel->findOpening(0, $endDay, $rate);
+            if (!$openings) {
                 throw new ContactClientException(
-                    'Could not find an open time slot to send in the next '.$seekDays.' days',
+                    'Could not find an open time slot to send in the next '.$endDay.' days',
```

Comment:
I'd consider replacing this with double quotes just for readability with $endDay